src/containers/Tenant/Diagnostics/TenantOverview/TabCard/TabCard.tsx

```diff
@@ -15,15 +15,53 @@ interface TabCardProps {
     active?: boolean;
     helpText?: string;
     legendFormatter: (params: {value: number; capacity: number}) => string;
+    variant?: 'default' | 'serverless';
+    subtitle?: string;
 }
 
-export function TabCard({text, value, limit, active, helpText, legendFormatter}: TabCardProps) {
+export function TabCard({
+    text,
+    value,
+    limit,
+    active,
+    helpText,
+    legendFormatter,
+    variant = 'default',
+    subtitle,
+}: TabCardProps) {
     const {status, percents, legend, fill} = getDiagramValues({
         value,
         capacity: limit,
         legendFormatter,
     });
 
+    if (variant === 'serverless') {
+        return (
+            <div className={b({active})}>
+                <Card
+                    className={b('card-container', {active})}
+                    type="container"
+                    view={active ? 'outlined' : 'raised'}
+                >
+                    <div className={b('legend-wrapper')}>
```

Comment:
It seems only card content varies depend on `variant`, while wrapper is same. Lets make `renderContent` function?